### PR TITLE
fix(conf): make a better style alignment in lv_conf_internal.h

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

and reorder Kconfig and src/lv_conf_kconfig.h as lv_conf_template.h.
Note: the behavior should be same as before.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: lv_conf_template.h [hunk 1]
```diff
@@ -183,21 +183,21 @@ e.g. "stm32f769xx.h" or "stm32f429xx.h"*/
...
-#  define LV_LOG_LEVEL    LV_LOG_LEVEL_WARN
+#  define LV_LOG_LEVEL LV_LOG_LEVEL_WARN
 
 /*1: Print the log with 'printf';
  *0: User need to register a callback with `lv_log_register_print_cb()`*/
-#  define LV_LOG_PRINTF   0
+#  define LV_LOG_PRINTF 0
 
 /*Enable/disable LV_LOG_TRACE in modules that produces a huge number of logs*/
-#  define LV_LOG_TRACE_MEM            1
```
Comment (kisvegabor) [inline lv_conf_template.h:193]: Personally, I found the numbers aligned in one column more readable. 
As this file is for humans, I think readability is important. What do you think?

Comment (xiaoxiang781216) [reply lv_conf_template.h:193]: Ok, let's keep the related macros align each other, but remove the duplicated spaces from other macros.

Comment (kisvegabor) [reply lv_conf_template.h:193]: Ok.

Comment (xiaoxiang781216) [reply lv_conf_template.h:193]: Done.

**Comment by kisvegabor** [comment]
Looks good, thank you!

**Pull request merged**